src/components/toDoController.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable import/no-cycle */
+import { format } from 'date-fns';
+import { lightBackground } from './displayEffects';
+
+
+import { renderLists, renderListContent } from './renderDom';
+
+import { removeBox } from './renderPopUpBox';
+
+
+const toDoFactory = (toDos,
+  list,
+  notes,
+  id,
+  active = false) => ({
+  toDos, list, notes, id, active,
+});
+
+export const toDoArray = [];
+
+toDoArray.push(toDoFactory([{
+  title: 'Create the functionality and the front end part',
+  dueDate: '02/03/2021',
+  done: false,
+  priority: false,
+},
+{
+  title: 'use webpack to compile the code',
+  dueDate: '10/03/2021',
+  done: false,
+  priority: true,
+}], 'Microverse', 'Microverse wants student to create a todo App', '1609618397810', true));
+
+export function getInput(id) {
+  return document.getElementById(id).value;
+}
+
+
+export function addNewToDo() {
+  let date = '';
+
+  if (getInput('box-date') !== '') {
+    date = format(new Date(getInput('box-date')), 'dd/MM/yyyy');
+  }
+
+  /* eslint-disable no-console */
+
+  for (let i = 0; i < toDoArray.length; i += 1) {
+    if (toDoArray[i].active === true) {
+      toDoArray[i].toDos
+        .push({
+          title: getInput('box-title-input'),
+          dueDate: date,
+          done: false,
+          priority: document.querySelector('#box-prio-checkbox').checked,
+        });
+      renderListContent();
+      lightBackground();
+      removeBox();
+      console.log(toDoArray[i].toDos);
+    }
+  }
+
+  date = '';
+}
+/* eslint-enable no-console */
+
+export function setActiveList(targetListId) {
```

Comment:
- [ ] Please change this and **all** other regular functions to es6 arrow function syntax.